ansible_collections/arista/avd/molecule/eos_designs_unit_tests/custom_modules/custom_interface_descriptions_with_data.py

```diff
@@ -32,7 +34,8 @@ def underlay_ethernet_interface(self, data: InterfaceDescriptionData) -> str:
         if data.link_type == "underlay_l2":
             return f"{self._custom_description_prefix}_{link_peer}_{data.peer_interface}"
 
-        return ""
+        elems = [data.wan_carrier, data.wan_circuit_id, data.peer, data.peer_interface]
+        return "_".join([elem for elem in elems if elem])
```

Comment:
not sure if this is really needed here but thats ok

Reply:
I have asked for these changes to be removed.